docs/docs/hpc-on-k8s/tensorflow.md

```diff
@@ -0,0 +1,56 @@
+# TensorFlow Workload
+TensorFlow is an open-source end-to-end platform for machine learning. It has a comprehensive, flexible ecosystem of tools, libraries, and community resources that lets researchers push the state-of-the-art in ML and developers easily build and deploy ML-powered applications. 
+
+## Steps to Execute Job
+Below is the example manifest file for TensorFlow workload in a Kubernetes cluster.
+```yaml
+# job.yaml
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: tensorflow-
+  namespace: default
+  labels:
+    kueue.x-k8s.io/queue-name: user-queue
+spec:
+  parallelism: 1
+  completions: 1
+  suspend: true
+  template:
+    spec:
+      containers:
+      - name: dummy-job
+        image: shubhamdang/custom_tensorflow:1.15.5
```

Comment:
Is this still the image repository or has the openflight one been created now?

Reply:
updated and image is prsent in openflighthpc dockerhub account.